pr_line_related_po_line/views/purchase_order_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<odoo>
+    <record model="ir.ui.view" id="purchase_order_form">
+        <field name="name">purchase.order.prl.related.pol.inherit.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/tree/field[@name='invoice_lines']" position="after">
+                <field name="purchase_requisition_line_id"
+                       invisible="1"
+                       force_save="1"/>
```

Comment:
`force_save` has effect only on readonly fields, which seems to not be the case.

Same below.

Reply:
Done 😄